addons/website_sale_coupon/views/website_sale_templates.xml

```diff
@@ -75,6 +75,12 @@
     </xpath>
 </template>
 
+<template id="display_coupon_form" inherit_id="website_sale.total">
```

Comment:
`open_coupon_form` being in `website_sale.reduction_code`, it should inherit from that view, not `website_sale.total`.

Otherwise it will just crash when you archive the `website_sale.reduction_code` view as the `open_coupon_form` won't be found in `website_sale.total`.

And as `website_sale.reduction_code` is a customize show view, that's even more critic

Reply:
@rdeodoo i've changed code as per your feedback